cmd/jimmsrv/service/service.go

```diff
@@ -273,6 +278,12 @@ func NewService(ctx context.Context, p Params) (*Service, error) {
 	s := new(Service)
 	s.mux = chi.NewRouter()
 
+	// chi middlewares
+	if p.LogLevel == "debug" {
+		s.mux.Use(chimiddleware.RequestLogger(&logger.HTTPLogFormatter{}))
+	}
+	s.mux.Use(chimiddleware.Recoverer)
```

Comment:
also, let's please not do this in this PR.. this change has nothing to do with logging and might feel like you're trying to sneak in a change ;)

Reply:
fair enough